resources/assets/sass/slideshow.sass

```
@@ -161,6 +161,17 @@ $border-color: $color-inactive-gray
 			color: $color-gray
 			font-size: $font-size-minus-2
 
+	.annotation-meta-signature
```

```suggestion
	.comment-meta-signature
```
The rest of classes will be updated in https://github.com/bethinkpl/wnl-platform/pull/1455.